samples/RXPTest/src/Tests/ViewBasicTest.tsx

```diff
@@ -106,6 +110,87 @@ class BasicView extends RX.Component<RX.CommonProps, RX.Stateless> {
                           'I have differing border radii' }
                     </RX.Text>
                 </RX.View>
+
+                {/* View's accessibilityLabel (both explicit and automatic recursive generation)
+                  * and importantForAccessibility all possible values and hierarchical combinations. */}
+
+                <RX.View 
+                    style={ _styles.explainTextContainer } 
+                    key={ 'explanation2' }
+                >
+                    <RX.Text style={ _styles.explainText }>
+                        { 'Should be read by screen reader as a single text element \"Alpha Beta Gamma Delta\" ' + 
+                          'and screen reader should not be able to get to individual texts.' }
+                    </RX.Text>
+                </RX.View>
+                <RX.View style={ _styles.labelContainer }>
+                    { BasicView.accessibilityLabelAndImportantForAccessibilityTestUI(RX.Types.ImportantForAccessibility.Yes) }
+                </RX.View>
+
+                <RX.View 
+                    style={ _styles.explainTextContainer } 
+                    key={ 'explanation3' }
+                >
+                    <RX.Text style={ _styles.explainText }>
+                        { 'Should be read by screen reader as four separate text elements: ' + 
+                          '\"Alpha\", \"Beta\", \"Gamma\", \"Delta\".' }
+                    </RX.Text>
+                </RX.View>
+                <RX.View style={ _styles.labelContainer }>
+                    { BasicView.accessibilityLabelAndImportantForAccessibilityTestUI(RX.Types.ImportantForAccessibility.No) }
+                </RX.View>
+            </RX.View>
+        );
+    }
+
+    private static accessibilityLabelAndImportantForAccessibilityTestUI(important: RX.Types.ImportantForAccessibility) {
```

Comment:
This could be a free-floating function in this file rather than a `static` method.